insights/specs/__init__.py

```diff
@@ -361,6 +361,8 @@ class Specs(SpecSet):
     ovirt_engine_console_log = RegistryPoint(filterable=True)
     ovirt_engine_server_log = RegistryPoint(filterable=True)
     ovirt_engine_ui_log = RegistryPoint(filterable=True)
+    ovs_appctl_fdb_show_bridge = RegistryPoint(multi_output=True)
+    ovs_vsctl_list_br = RegistryPoint()
```

Comment:
See comment below.

Reply:
@bfahr - As per suggestion, PR is updated to include the changes from other PRs.